app/assets/stylesheets/mail/email.scss

```diff
@@ -107,6 +109,49 @@ table.order-summary {
   }
 }
 
+// put in separate order invoice scss file?
+table.payment-list {
+  width: 100%;
```

Comment:
"should we create a separate scss file for the invoice styles?"

yes, I think this should go to a separate file :+1:

I'd keep the reference to emails in the name as invoices are only using email partials. Unless you want to create partials that are specific to invoices.